dev/benchmarks/complex_layout/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -166,18 +180,33 @@
 /* End PBXResourcesBuildPhase section */
 
 /* Begin PBXShellScriptBuildPhase section */
-		9740EEB61CF901F6004384FC /* ShellScript */ = {
+		3B06AD1E1E4923F5004D2608 /* Thin Binary */ = {
 			isa = PBXShellScriptBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
 			);
 			inputPaths = (
 			);
+			name = "Thin Binary";
 			outputPaths = (
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "/bin/sh $FLUTTER_ROOT/packages/flutter_tools/bin/xcode_backend.sh";
```

Comment:
This was what caused the devicelab errors on #64988 post-submit.  `xcode_backend.sh` without any args was building, thinning, and embedding.  However, at this point the generated Info.plist that we will need to edit for https://github.com/flutter/flutter/pull/64988 doesn't exist, so it was failing.